### PR TITLE
prepare to release 1.22.0

### PR DESCRIPTION
**Pull request by cinchurge**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+## [1.21.3] - 2021-08-13
```
Comment (prashantv) [inline CHANGELOG.md:4]: let's do a minor version bump, since a new API in the channel options (`SkipHandlerMethods`)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+### Changed
+
+ * Add options to support handlers with ignoring methods
```
Comment (prashantv) [inline CHANGELOG.md:7]:
```suggestion
 * Add `SkipHandlerMethods` option as an allow-list of metohds that are handled by the override handler
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+ * Allow method registration if handler implements Register
+ * Remove Travis CI, add Github Actions, bypass 2 flaky tests in GHA
```
Comment (prashantv) [inline CHANGELOG.md:9]: no need to put this in the changelog

File: CHANGELOG.md (hunk 2 above)
Comment (prashantv) [reply CHANGELOG.md:7]: this should go under `Added` since it's a new feature

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+## [1.22.0] - 2021-08-13
+### Changed
+
+ * Add `SkipHandlerMethods` option as an allow-list of metohds that are handled by the override handler.
```
Comment (prashantv) [inline CHANGELOG.md:7]: This should be unded a `Added` heading.

**Review by prashantv — APPROVED** [review]

**Review by prashantv — APPROVED** [review]

**Review by AllenLuUber — APPROVED** [review]

**Pull request merged**